lib/api/request/requestResponse.ts

```diff
@@ -42,6 +42,8 @@ export class Headers {
       get: (target, name) => this.getHeader(name as string),
       set: (target, name, value) => this.setHeader(name as string, value),
     });
+
+    this.setHeader('X-Kuzzle-Node', (global as any).kuzzle.id);
```

Comment:
Can't this be as string ?

Reply:
`kuzzle.id` is a string but I have to cast the `global` because the `kuzzle` property is not defined on it

Reply:
This won't be necessary once #1913 gets merged, `global.kuzzle` will be accessible directly from TS scripts.